keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -98,11 +99,26 @@ class ClusterHandler(
           imageService.getLatestNamedImage(artifactName, "test")
         } ?: throw NoImageFound(artifactName)
 
-        log.info("Images found for {}: {}", artifactName, namedImage)
+        log.info("Image found for {}: {}", artifactName, namedImage)
 
         val amis = namedImage.amis[region] ?: throw NoImageFoundForRegion(artifactName, region)
         return amis.first() // todo eb: when are there multiple?
       }
+      is JenkinsJobImageProvider -> {
+        val namedImage = runBlocking {
+          imageService.getNamedImageFromJenkinsInfo(
+            imageProvider.packageName,
+            "test",
```

Comment:
This should be a dynamic property that just defaults to test (same with other occurrences in the class.)